internal/lib/lib.go

```diff
@@ -1004,8 +1004,8 @@ func GetPassthroughShardVSName(s string, key string) string {
 	shardVsPrefix := GetClusterName() + "--" + PassthroughPrefix
 	vsNum = utils.Bkt(s, shardSize)
 	vsName := shardVsPrefix + strconv.Itoa(int(vsNum))
```

Comment:
We need to "EVH" in vsname . For example, "cluster1--Shared-Passthrough-EVH-0" For SNI, it will be "cluster1--Shared-Passthrough-0" like that.

Reply:
Passthrough VSes don't depend on EVH, so I thought the naming can be same .. similar to L4 VS.